src/utils/dates.js

```diff
@@ -10,3 +10,9 @@ export const ONE_DAY_IN_MILLISECONDS = 86400000;
 export function getFutureDate(offset) {
 	return new Date(Date.now() + offset * ONE_DAY_IN_MILLISECONDS);
 }
+
+export function getDaysBetweenDates(dateOne, dateTwo) {
```

Comment:
I would suggest changing the names of the arguments to 'startDate' and 'endDate' to enhance readability.

Reply:
Agreed. Parameter names have been changed.